test/test_utils.ml

```diff
@@ -103,6 +103,51 @@ let util_safe_html _ =
     {|<a href="localhost:2318/foo_w?lang=fr&#38;image=on">foo</a>|}
     (Util.safe_html {|<a href="localhost:2318/foo_w?lang=fr&image=on">foo</a>|})
 
+(* si on savait construire conf, on testerait directement transl_a_of_b *)
```

Comment:
You can use https://github.com/geneweb/geneweb/pull/975, build your own lexicon, and avoid exposing internal functions for testing purpose.

NB: If you expose internal functions, it would be better to specify in documentation that the function is not meant to be used outside of this module.

```
(**/**)
(** Internal function, only exposed for testing purpose. You should not use it directly. *)
(**/**)

```